x-pack/legacy/plugins/siem/public/components/url_state/test_dependencies.ts

```diff
@@ -5,28 +5,27 @@
  */
 
 import { ActionCreator } from 'typescript-fsa';
-import { hostsModel, networkModel } from '../../store';
-import { UrlStateContainerPropTypes, LocationTypes, KqlQuery } from './types';
-import { CONSTANTS } from './constants';
+import { Query } from 'src/plugins/data/common';
+
 import { DispatchUpdateTimeline } from '../open_timeline/types';
 import { navTabs, SiemPageName } from '../../pages/home/home_navigations';
-import { hostsActions, inputsActions, networkActions } from '../../store/actions';
+import { hostsModel, networkModel } from '../../store';
+import { inputsActions } from '../../store/actions';
 import { HostsTableType } from '../../store/hosts/model';
+
+import { CONSTANTS } from './constants';
 import { dispatchSetInitialStateFromUrl } from './initialize_redux_by_url';
+import { UrlStateContainerPropTypes, LocationTypes } from './types';
 
 type Action = 'PUSH' | 'POP' | 'REPLACE';
 const pop: Action = 'POP';
 
-export const getFilterQuery = (queryLocation: LocationTypes): KqlQuery => ({
-  filterQuery: {
-    expression: 'host.name:"siem-es"',
-    kind: 'kuery',
-  },
-  queryLocation,
+export const getFilterQuery = (): Query => ({
+  query: 'host.name:"siem-es"',
+  language: 'kuery',
 });
 
-export const mockApplyHostsFilterQuery: jest.Mock = (hostsActions.applyHostsFilterQuery as unknown) as jest.Mock;
-export const mockApplyNetworkFilterQuery: jest.Mock = (networkActions.applyNetworkFilterQuery as unknown) as jest.Mock;
+export const mockSetFilterQuery: jest.Mock = (inputsActions.setFilterQuery as unknown) as jest.Mock;
```

Comment:
I think these types might be able to be cleaned up with something similar to what is mentioned [here](https://stackoverflow.com/a/52366601)?

Reply:
I will try to re-visit this comment in another time